README.md

```diff
@@ -1 +1,37 @@
 # Gemstock-Mk2
+
+A delta 3D printer designed by 3 enthousiasts with cost and effectiveness.
+
```

Comment:
A delta 3D printer designed by 3 enthousiasts with cost and effectiveness **in mind** zou ik zeggen